### PR TITLE
Close #684 Heating Analyser for Multi Species

### LINKED ISSUE #684: Multi species support for numerical heating analyzer
Extend the numerical heating analyzer tool `plotNumericalHeating` to work with any number of particles.

(follow up to #672 - as suggested by @ax3l )

### PR DESCRIPTION
**Pull request by PrometheusPi**
With this pull request the `plotNumericalHeating` tool  supports any number of species (#684).

It goes through the `simOutput` directory and looks for `Energy*.dat` files with first row *time* and second row *energy*. 

I've tested it with some nonsense files and it worked fine.

### CODE REVIEW

**Comment by ax3l** [comment]
:+1: 
and look at the stats: `+49 −65` told ya :)

File: src/tools/bin/plotNumericalHeating [hunk 1]
```diff
@@ -82,88 +85,69 @@ args = parser.parse_args()
...
 
 
 ## check if directories exit ###
 simDir = "/simOutput/"
 
-warningText1 = "The {} directory ({}) does not exist." 
-if not os.path.isdir(directory1):
-    sys.exit(warningText1.format("first", directory1))
+warningText1 = "The directory number {} does not exist ({})." 
+warningText2 = "The directory number {} does not contain {} yet."
```
Comment (ax3l) [inline src/tools/bin/plotNumericalHeating]: I would put (yet) in brackets :) maybe it will never exist ;)

**Comment by ax3l** [comment]
@PrometheusPi looks ok to me, maybe a minor update in one of the error messages? :)

**Comment by PrometheusPi** [comment]
@ax3l I will update it :)

**Pull request merged**